tools/README.md

```diff
@@ -0,0 +1,58 @@
+## Deployment tool
+
+There is a JS library under `tools/deployment_tool`, which helps in building deployment scripts for non-traditional scenarios.
+
+### Features
+
+- Can output the `accounts` section for a [Parity chainspec][parity-chainspec] to allow
```

Comment:
```
 - Can output the `accounts` section for a [Parity chainspec][parity-chainspec] to allow
```
or
```suggestion
- Can output the `accounts` section for a [Parity chainspec](parity-chainspec) to allow
```

(compare line [43](https://github.com/OpenSTFoundation/mosaic-contracts/pull/458/files#diff-07b723ae57aaa00bce72a3022cf09ecdR43))

Reply:
Should be []